### PR TITLE
Set up Jest framework and create tests for pure, stateless functions

### PR DESCRIPTION
**Pull request by abassey2**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Sets up Jest testing framework, updates documentation, and creates test for pure, stateless function

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/233
- References https://github.com/retaildevcrews/wcnp/issues/215

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (aflinchb) [inline client/.env]: Just curious, why is this needed? Or was it a default setting/file?

File: client/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-react',
+    ],
```
Comment (aflinchb) [inline client/babel.config.js:5]: I see a few of those pesky "no newline at end of file". I'll do a search to see how or if we can get that configured in eslint since right now it's a non blocking thing that pops up in PRs.

I see it in .env, this configuration, stylesMock.js, and jest.config.js.

**Review by aflinchb — APPROVED** [review]
Approved w/ 2 comments/questions 👍👍

Another question I had is if it was discussed yet how yell want to structure (directory/folder-wise) where the tests will live. I don't think it needs to be answered for this PR, but figured I'd ask :)

File: client/.env (hunk 1 above)
Comment (abassey2) [reply client/.env]: This was needed in order for npm start to work due to that dependency issue I was running into. I tried deleting the node modules and reinstalling and also following all the listed steps and neither of those were working. This was the only solution I could find.

**Comment by abassey2** [comment]
Johee and I decided to have all the tests live within the component/function that it's testing's folder for now!

File: client/babel.config.js (hunk 2 above)
Comment (abassey2) [reply client/babel.config.js:5]: I have been looking into it and the linter should be picking it up, but I am unsure why it is not

Comment (abassey2) [reply client/babel.config.js:5]: I added a newline at the end of all the files, but for some reason I still get the no new line at EOF for babel.config.js and jest.config.js

Comment (abassey2) [reply client/babel.config.js:5]: ^ fixed this
